tests/tests_pytorch/checkpointing/test_model_checkpoint.py

```diff
@@ -1320,6 +1375,23 @@ def test_save_last_versioning(tmpdir):
     assert {"last.ckpt", "last-v1.ckpt"} == set(os.listdir(tmpdir))
 
 
+def test_save_last_versioning_disabled(tmpdir):
```

Comment:
This test could be removed in favor of also adding `save_last=True` to `test_ckpt_version_counter_disabled_after_rerun_new_trainer`